templates/element_doc_pages/any-page.html

```diff
@@ -156,16 +156,16 @@ <h3>Add Custom (S)CSS</h3>
       </xmp>
     </blockquote>
     <p>
-      Then refresh your new page <a href="/pages/spotlight_on?subject=mustard">/pages/spotlight_on?subject=mustard</a>
```

Comment:
Ah, farewell mustard :)

Reply:
yeah - it was sad to replace "mustard" with boring, generic "tissue".